tsup.config.ts

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from 'tsup';
+
+export default defineConfig({
+  entry: ['lib/index.ts'],
+  splitting: false,
+  sourcemap: true,
+  clean: true,
```

Comment:
[note] This didn't work as expected for me. After building with `tsup` for the first time, my `dist` folder had some leftovers like `dist/cjs/utils/<empty>`

Reply:
Hm, we could introduce an additional prebuild step and use `rimraf` but it seems like an overkill? What do you think?

Reply:
I am only concerned whether we might _npm publish_ leftovers at some point if this it not working as intended. Your suggestion with `rimraf` would work as well, but I am also not strong either. Better safe than sorry, and also you account for all cases, even if we pick up another bundler in the future that does not support `clean` feature at all. Up to you how to proceed with this.

Reply:
This is not going to happen as part of the automated release pipeline, because dist/cjs doesn't exist at this point.

Reply:
I was mostly referring to cases where someone publishes from their local machine. Resolving...